src/universal/modules/userDashboard/components/OrgMembers/OrgMembers.js

```diff
@@ -1,221 +1,24 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import {connect} from 'react-redux';
 import {createPaginationContainer} from 'react-relay';
-import {withRouter} from 'react-router-dom';
-import {Button, Panel, Toggle, Tooltip} from 'universal/components';
-import {Menu, MenuItem} from 'universal/modules/menu';
-import {showError, showInfo} from 'universal/modules/toast/ducks/toastDuck';
-import LeaveOrgModal from 'universal/modules/userDashboard/components/LeaveOrgModal/LeaveOrgModal';
-import RemoveFromOrgModal from 'universal/modules/userDashboard/components/RemoveFromOrgModal/RemoveFromOrgModal';
-import InactivateUserMutation from 'universal/mutations/InactivateUserMutation';
-import SetOrgUserRoleMutation from 'universal/mutations/SetOrgUserRoleMutation';
-import {BILLING_LEADER, PERSONAL} from 'universal/utils/constants';
-import withMutationProps from 'universal/utils/relay/withMutationProps';
-import OrgUserRow from '../OrgUserRow/OrgUserRow';
-import styled from 'react-emotion';
-import ui from 'universal/styles/ui';
-
-const originAnchor = {
-  vertical: 'top',
-  horizontal: 'right'
-};
-
-const targetAnchor = {
-  vertical: 'top',
-  horizontal: 'left'
-};
-
-
-const ActionsBlock = styled('div')({
-  alignItems: 'center',
-  display: 'flex',
-  justifyContent: 'flex-end'
-});
-
-const MenuToggleBlock = styled('div')({
-  marginLeft: ui.rowGutter,
-  width: '1.5rem'
-});
-
-const ToggleBlock = styled('div')({
-  marginLeft: ui.rowGutter,
-  width: '6.25rem'
-});
+import {Panel} from 'universal/components';
+import OrgMemberRow from 'universal/modules/userDashboard/components/OrgUserRow/OrgMemberRow';
 
 const OrgMembers = (props) => {
   const {
-    dispatch,
-    history,
-    org,
-    orgId,
-    viewer: {organization: {orgMembers}},
-    relay: {environment},
-    submitMutation,
-    onError,
-    onCompleted
+    viewer: {organization}
   } = props;
-
-  const {tier} = org;
-  const isPersonalTier = tier === PERSONAL;
-  const setRole = (userId, role = null) => () => {
-    const variables = {orgId, userId, role};
-    SetOrgUserRoleMutation(environment, variables, {dispatch, history});
-  };
-
-  const viewerIsBillingLeader = true;
-
+  const {orgMembers} = organization;
   const billingLeaderCount = orgMembers.edges.reduce((count, {node}) => node.isBillingLeader ? count + 1 : count, 0);
-
-  const userRowActions = (orgMember) => {
```

Comment:
the actions are a function of the menu, not the row & not the entire page, so i moved them down the tree.